src/modules/person/api/lock-user.body.params.ts

```diff
@@ -16,4 +16,11 @@ export class LockUserBodyParams {
         nullable: false,
     })
     public readonly locked_from!: string;
+
+    @IsDate()
+    @ApiProperty({
+        required: true,
+        nullable: true,
+    })
+    public readonly locked_until!: Date;
```

Comment:
`required` and `nullable` are contradictory. 
If it really is an optional property, you should remove the `required`, add another `IsOptional`-decorator and make sure the type is `Date | undefined`

Reply:
adjusted